src/_base/helm/app/templates/application/webapp/ingress.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if eq .Values.ingress.type "standard" }}
+{{ if and (.Values.docker.services.nginx.enabled) (eq .Values.ingress.type "standard") }}
```

Comment:
Needs a check for varnish as well as nginx due to `.Values.ingress.target_service` target. Whilst doubtful you'd have varnish and no nginx, varnish's VCL can be doing anything.